ci/scripts/release.sh

```diff
@@ -124,23 +124,26 @@ if [[ -n "${BUILDKITE_TAG}" ]]; then
 
     echo "--- Release upload risingwave asset"
     tar -czvf risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz risingwave
-    gh release upload "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
+    gh release upload --clobber "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-"${ARCH}"-unknown-linux.tar.gz
```

Comment:
I think it's unnecessary for artifacts with `ARCH` in their names?

Reply:
In some rare cases, we can rerun the pipeline without deleting the assets with `--clobber`. Otherwise we have to delete the assets manually before rerunning the pipeline.